### PR TITLE
[TCD-2117] Remove all TestObject refs

### PR DESCRIPTION
**Pull request by sweeneyskirt-sl**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/basics/acct-team-mgmt/test-object.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-<p><span className="sauceGold">DEPRECATED</span></p>
+<p><span className="sauceRed">REMOVED</span></p>
```
Comment (linafarid) [inline docs/basics/acct-team-mgmt/test-object.md:10]: Hi @sweeneyskirt-sl ,
Can you let me know the reason we mention TO as REMOVED and not completely erase it from the docs?

File: docs/mobile-apps/automated-testing/appium/appium-migration.md [hunk 2]
```diff
@@ -2,164 +2,12 @@
 id: migration
 title: Migrating to Appium on Sauce
 sidebar_label: Legacy Migration
-description: Convert your TestObject Appium setup to Sauce Labs testing.
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-<p><span className="sauceGold">DEPRECATED</span></p>
+<p><span className="sauceRed">REMOVED</span></p>
```
Comment (linafarid) [inline docs/mobile-apps/automated-testing/appium/appium-migration.md:9]: same question as above. Is there a reason why we want to keep this page at all?

File: docs/basics/acct-team-mgmt/test-object.md (hunk 1 above)
Comment (sweeneyskirt-sl) [reply docs/basics/acct-team-mgmt/test-object.md:10]: Hey @linafarid -- At this time, we are still working on our internal docs redirect solution, so we have a freeze on physical removal of any page in the docs that has been published to production, and is, therefore, indexed. If anyone has linked to this page externally (including our own marketing site or the Sauce Labs app), we would rather they hit this page and have an explanation that hit a 404. I have removed this page from our navigation, as well as all links to it from our docs, so the only way someone would reach it is by clicking an external link or entering the URL manually.

**Review by linafarid — APPROVED** [review]

**Review by kimsauce — COMMENTED** [review]
LGTM! Thanks for doing.

**Pull request merged**